odl_video/test_settings.py

```diff
@@ -0,0 +1,5 @@
+"""
+Django test settings for odl_video project.
+"""
+from odl_video.settings import *  # pylint: disable=wildcard-import,unused-wildcard-import
+USWITCH_URL = 'https://testing_odl.mit.edu'
```

Comment:
Do you have an example of that? I originally tried the following in `conftest.py`:
```
def pytest_configure():
    settings.configure(USWITCH_URL='https://testing_odl.mit.edu')
```

but then all the tests failed because 'settings were already configured', in pytest.ini: `ds=odl_video.settings`